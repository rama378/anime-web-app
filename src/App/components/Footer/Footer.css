.Footer {
    border-top: 1px solid rgba(255, 255, 255, 0.3);
    margin: 16px;
    padding: 16px;
    text-align: right;
}
