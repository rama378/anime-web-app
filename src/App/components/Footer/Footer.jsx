import { APP_NAME } from '../../common/constants/constant';
import { getCurrentYear } from '../../common/utils/dateUtils';
import './Footer.css';

const Footer = () => {
	return (
		<div className='Footer'>
			@ {getCurrentYear()} {APP_NAME}
		</div>
	);
};

export default Footer;
