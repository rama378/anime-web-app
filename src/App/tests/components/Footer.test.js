import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Footer from '../../components/Footer/Footer';

const renderer = new ShallowRenderer();

describe('<Footer />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<Footer />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
