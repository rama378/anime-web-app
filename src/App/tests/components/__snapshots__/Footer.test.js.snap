// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render and match the snapshot 1`] = `
<div
  className="Footer"
>
  @ 
  2023
   
  Anime Web App
</div>
`;
